src/Sulu/Component/Rest/Exception/DependantResourcesFoundExceptionInterface.php

```diff
@@ -15,6 +15,11 @@
 
 interface DependantResourcesFoundExceptionInterface extends RestExceptionInterface
 {
+    /**
+     * @return array{id: int|string, resourceKey: string}
+     */
+    public function getResource(): array;
+
     /**
      * @return array<int, array<array{id: int|string, resourceKey: string}>>
```

Comment:
woudl call this `dependantResourceBatches` to make clear that it is an array of arrays of resources